src/ErrorHandling/RecentErrorsPage.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from "react";
+import { makeStyles } from "tss-react/mui";
+import { type ErrorRecord, ErrorState } from "./ErrorState";
+import { useRerender } from "../ui/React/hooks";
+import { Typography, Tooltip } from "@mui/material";
+import { Theme } from "@mui/material/styles";
+
+export function RecentErrorsPage(): React.ReactElement {
+  const rerender = useRerender();
+  React.useEffect(() => {
+    const clearSubscription = ErrorState.ErrorUpdate.subscribe(rerender);
+    ErrorState.UnreadErrors = 0;
+    return () => {
+      clearSubscription();
+    };
```

Comment:
```suggestion
    return () => {
      clearSubscription();
      ErrorState.UnreadErrors = 0;
    };
```
You have to reset `ErrorState.UnreadErrors` when this component is unmounted.